services/wiki/src/db/knexTypes.ts

```diff
@@ -135,3 +135,5 @@ export type TCleanTextOptions = {
 export type TCleanHFResponseOutput = {
   generated_text: string
 }[]
+
+export type TSupportedLanguage = 'en' | 'es' | 'ca'
```

Comment:
Recomendación usar enums para tener validación en runtime:

```suggestion
export enum SupportedLanguage {
  English = 'en',
  Spanish = 'es',
  Catalan = 'ca',
}```